### ISSUE TITLE
New submissions for Mon, 30 Oct 23

### ISSUE THREAD

**Issue by olozhika** [olozhika/ArXivDaily_StarFormation#135]
# New submissions for Mon, 30 Oct 23
Auto update Star Formation & Molecular Cloud papers at about 2:30am UTC (10:30am Beijing time) every weekday.

阅读 `Usage.md`了解如何使用此repo实现个性化的Arxiv论文推送

See `Usage.md` for instructions on how to personalize the repo. 

Keyword list: ['star formation', 'molecular cloud', 'interstellar medium', 'cloud', 'clump', 'core', 'filament', 'atomic gas', 'N-PDF']

Excluded: ['galaxies', 'galaxy cluster', 'AGN']

#### Feature Extraction and Classification from Planetary Science Datasets  enabled by Machine Learning
 - **Authors:** Conor Nixon, Zachary Yahn, Ethan Duncan, Ian Neidel, Alyssa Mills, Benoît Seignovert (OSUNA), Andrew Larsen, Kathryn Gansler, Charles Liles, Catherine Walker, Douglas Trent, John Santerre
 - **Subjects:** Earth and Planetary Astrophysics (astro-ph.EP); Instrumentation and Methods for Astrophysics (astro-ph.IM); Machine Learning (cs.LG)
 - **Arxiv link:** https://arxiv.org/abs/2310.17681
 - **Pdf link:** https://arxiv.org/pdf/2310.17681
 - **Abstract**
 In this paper we present two examples of recent investigations that we have undertaken, applying Machine Learning (ML) neural networks (NN) to image datasets from outer planet missions to achieve feature recognition. Our first investigation was to recognize ice blocks (also known as rafts, plates, polygons) in the chaos regions of fractured ice on Europa. We used a transfer learning approach, adding and training new layers to an industry-standard Mask R-CNN (Region-based Convolutional Neural Network) to recognize labeled blocks in a training dataset. Subsequently, the updated model was tested against a new dataset, achieving 68% precision. In a different application, we applied the Mask R-CNN to recognize clouds on Titan, again through updated training followed by testing against new data, with a precision of 95% over 369 images. We evaluate the relative successes of our techniques and suggest how training and recognition could be further improved. The new approaches we have used for planetary datasets can further be applied to similar recognition tasks on other planets, including Earth. For imagery of outer planets in particular, the technique holds the possibility of greatly reducing the volume of returned data, via onboard identification of the most interesting image subsets, or by returning only differential data (images where changes have occurred) greatly enhancing the information content of the final data stream.
#### Gaps in the Main-Sequence of Star Cluster Hertzsprung Russell Diagrams
 - **Authors:** Priya Hasan
 - **Subjects:** Solar and Stellar Astrophysics (astro-ph.SR); Astrophysics of Galaxies (astro-ph.GA)
 - **Arxiv link:** https://arxiv.org/abs/2310.17725
 - **Pdf link:** https://arxiv.org/pdf/2310.17725
 - **Abstract**
 The presence of gaps or regions of small numbers of stars in the main sequence of the Hertzsprung Russell Diagram (HRD) of star clusters has been reported in literature. This is interesting and significant as it could be related to star formation and/or rapid evolution or instabilities. In this paper, using Gaia DR3 photometry and confirmed membership data, we explore the HRD of nine open clusters with reported gaps, identify them and assess their importance and spectral types.
#### Formation of giant planets around intermediate-mass stars
 - **Authors:** Heather Johnston, Olja Panic, Beibei Liu
 - **Subjects:** Earth and Planetary Astrophysics (astro-ph.EP); Solar and Stellar Astrophysics (astro-ph.SR)
 - **Arxiv link:** https://arxiv.org/abs/2310.17767
 - **Pdf link:** https://arxiv.org/pdf/2310.17767
 - **Abstract**
 To understand giant planet formation, we need to focus on host stars close to $1.7\ \rm M_{\odot}$, where the occurrence rate of these planets is the highest. In this initial study, we carry out pebble-driven core accretion planet formation modelling to investigate the trends and optimal conditions for the formation of giant planets around host stars in the range of $1{-}2.4\ \rm M_{\odot}$. We find that giant planets are more likely to form in systems with a larger initial disk radius; higher disk gas accretion rate; pebbles of $\sim$ millimeter in size; and birth location of the embryo at a moderate radial distance of $\sim 10$ AU. We also conduct a population synthesis study of our model and find that the frequency of giant planets and super-Earths decreases with increasing stellar mass. This contrasts the observational peak at $1.7\ \rm M_{\odot}$, stressing the need for strong assumptions on stellar mass dependencies in this range. Investigating the combined effect of stellar mass dependent disk masses, sizes, and lifetimes in the context of planet population synthesis studies is a promising avenue to alleviate this discrepancy. The hot-Jupiter occurrence rate in our models is $\sim 0.7{-}0.8\%$ around $1\ \rm M_{\odot}$ - similar to RV observations around Sun-like stars, but drastically decreases for higher mass stars.
#### The role of turbulence in high-mass star formation: Subsonic and  transonic turbulence are ubiquitously found at early stages
 - **Authors:** Chao Wang, Ke Wang, Feng-Wei Xu, Patricio Sanhueza, Hauyu Baobab Liu, Qizhou Zhang, Xing Lu, F. Fontani, Paola Caselli, Gemma Busquet, Jonathan C. Tan, Di Li, J. M. Jackson, Thushara Pillai, Paul T. P. Ho, Andrés E. Guzmán, Nannan Yue
 - **Subjects:** Astrophysics of Galaxies (astro-ph.GA); Solar and Stellar Astrophysics (astro-ph.SR)
 - **Arxiv link:** https://arxiv.org/abs/2310.17970
 - **Pdf link:** https://arxiv.org/pdf/2310.17970
 - **Abstract**
 Context. Traditionally, supersonic turbulence is considered to be one of the most likely mechanisms to slow down the gravitational collapse in dense clumps, thereby enabling the formation of massive stars. However, several recent studies have raised differing points of view based on observations carried out with sufficiently high spatial and spectral resolution. These studies call for a re-evaluation of the role turbulence plays in massive star-forming regions. Aims. Our aim is to study the gas properties, especially the turbulence, in a sample of massive star-forming regions with sufficient spatial and spectral resolution, which can both resolve the core fragmentation and the thermal line width. Methods. We observed NH3 metastable lines with the Very Large Array (VLA) to assess the intrinsic turbulence. Results. Analysis of the turbulence distribution histogram for 32 identified NH3 cores reveals the presence of three distinct components. Furthermore, our results suggest that (1) sub- and transonic turbulence is a prevalent (21 of 32) feature of massive star-forming regions and those cold regions are at early evolutionary stage. This investigation indicates that turbulence alone is insufficient to provide the necessary internal pressure required for massive star formation, necessitating further exploration of alternative candidates; and (2) studies of seven multi-core systems indicate that the cores within each system mainly share similar gas properties and masses. However, two of the systems are characterized by the presence of exceptionally cold and dense cores that are situated at the spatial center of each system. Our findings support the hub-filament model as an explanation for this observed distribution
#### Radio outburst from a massive (proto)star. II. A portrait in space and  time of the expanding radio jet from S255 NIRS3
 - **Authors:** R. Cesaroni, L. Moscadelli, A. Caratti o Garatti, J. Eisloeffel, R. Fedriani, R. Neri, T. Ray, A. Sanna, B. Stecklum
 - **Subjects:** Astrophysics of Galaxies (astro-ph.GA)
 - **Arxiv link:** https://arxiv.org/abs/2310.18002
 - **Pdf link:** https://arxiv.org/pdf/2310.18002
 - **Abstract**
 Observations indicate that the accretion process in star formation may occur through accretion outbursts. This phenomenon has also now been detected in a few young massive (proto)stars (>8 Msun). The recent outburst at radio wavelengths of the massive (proto)star S255 NIRS3 has been interpreted by us as expansion of a thermal jet, fed by the infalling material. To follow up on our previous study and confirm our interpretation, we monitored the source for more than 1 yr in six bands from 1.5 GHz to 45.5 GHz and, after ~1.5 yr, with the Atacama Large Millimeter/submillimeter Array at two epochs, which made it possible to detect the proper motions of the jet lobes. The prediction of our previous study is confirmed by the new results. The radio jet is found to expand, while the flux, after an initial exponential increase, appears to stabilise and eventually decline. The radio flux measured during our monitoring is attributed to a single NE lobe, However, from 2019 a second lobe has been emerging to the SW, probably powered by the same accretion outburst, although with a delay of at least a couple of years. Flux densities at >6 GHz were satisfactorily fitted with a jet model, whereas those below 6 GHz are clearly underestimated by the model. This indicates that non-thermal emission becomes dominant at long wavelengths. Our results suggest that thermal jets can be a direct consequence of accretion events, when yearly flux variations are detected. The end of the accretion outburst is mirrored in the radio jet, as ~1 yr after the onset of the radio outburst, the inner radius of the jet began to increase while the jet mass stopped growing, as expected if the powering mechanism of the jet is quenched. Our findings support a tight connection between accretion and ejection in massive stars, consistent with a formation process involving a disk-jet system similar to that of low-mass stars.
#### LHAASO J2108+5157 as a Molecular Cloud Illuminated by a Supernova  Remnant
 - **Authors:** A. M. W. Mitchell
 - **Subjects:** High Energy Astrophysical Phenomena (astro-ph.HE)
 - **Arxiv link:** https://arxiv.org/abs/2310.18007
 - **Pdf link:** https://arxiv.org/pdf/2310.18007
 - **Abstract**
 The search for Galactic PeVatrons - astrophysical accelerators of cosmic rays to PeV energies - has entered a new phase in recent years with the discovery of the first Ultra-High-Energy (UHE, $E>100$ TeV) gamma-ray sources by the HAWC and LHAASO experiments. Establishing whether the emission is leptonic or hadronic in nature, however, requires multiwavelength data and modelling studies. Among the currently known UHE sources, LHAASO J2108+5157 is an enigmatic source without clear association to a plausible accelerator, yet spatially coincident with molecular clouds. We investigate the scenario of a molecular cloud illuminated by cosmic rays accelerated in a nearby supernova remnant (SNR) as an explanation for LHAASO J2108+5157. We aim to constrain the required properties of the SNR as well as which of the clouds identified in the vicinity is the most likely association. We use a model for cosmic ray acceleration in SNRs, their transport through the interstellar medium and subsequent interaction with molecular material, to predict the corresponding gamma-ray emission. The parameter space of SNR properties is explored to find the most plausible parameter combination that can account for the gamma-ray spectrum of LHAASO J2108+5157. In the case that a SNR is illuminating the cloud, we find that it must be young ($<10$ kyr) and located within $40-60$ pc of the cloud. A SN scenario with a low Sedov time is preferred, with a maximum proton energy of 3 PeV assumed. No SNRs matching these properties are currently known, although an as yet undetected SNR remains feasible. The galactic CR sea is insufficient to solely account for the observed flux, such that a PeVatron accelerator must be present in the vicinity.
#### The impact of convective criteria on the properties of massive stars
 - **Authors:** Yves Sibony, Cyril Georgy, Sylvia Ekström, Georges Meynet
 - **Subjects:** Solar and Stellar Astrophysics (astro-ph.SR)
 - **Arxiv link:** https://arxiv.org/abs/2310.18139
 - **Pdf link:** https://arxiv.org/pdf/2310.18139
 - **Abstract**
 We study the differences between models computed with Ledoux and Schwarzschild criteria on the internal structure, evolutionary track in the Hertzsprung-Russell diagram (HRD), lifetimes, evolution of the surface abundances and velocities, and masses of the He and CO cores. We investigate the consequences on the nature of the supernova (SN) progenitors and the type of SN events, as well as on the yields of light elements. We also study the impact on the outputs of population synthesis models. Models with initial masses between 7 and 120 M$_\odot$ at solar metallicity ($Z$=0.014) and with an initial rotation equal to 0 or 0.4 times the critical velocity at the zero-age main sequence were computed with either the Schwarzschild or the Ledoux criterion until the end of the C-burning phase. Models with initial masses between 15 and 32 M$_\odot$ computed with the Schwarzschild criterion show larger intermediate convective zones attached to the H-burning shell than models computed with the Ledoux criterion. Their CO cores and outer convective zones in the red supergiant (RSG) phase are also smaller. This impacts many outputs of stars during the core He-burning phase. Schwarzschild models have smaller CO cores and outer convective zones in the RSG phase, and their blue-to-red supergiant ratio is much higher than for Ledoux models. They also produce longer crossings of the Hertzsprung gap and favour blue loops. The upper luminosity of RSGs is little affected by the change in the convective criterion. The maximum luminosity of RSG progenitors for type II-P SN events is lowered from 5.2 to 4.95 when the Ledoux criterion is used instead of the Schwarzschild criterion in non-rotating models. The Schwarzschild criterion predicts longer-lasting, less nitrogen-enriched, and faster-rotating Cepheids. Rotational mixing decreases the differences between Schwarzschild and Ledoux models.
#### Molecular clouds in M51 from high-resolution extinction mapping
 - **Authors:** Helena Faustino Vieira, Ana Duarte-Cabral, Timothy A. Davis, Nicolas Peretto, Matthew W. L. Smith, Miguel Querejeta, Dario Colombo, Michael Anderson
 - **Subjects:** Astrophysics of Galaxies (astro-ph.GA)
 - **Arxiv link:** https://arxiv.org/abs/2310.18210
 - **Pdf link:** https://arxiv.org/pdf/2310.18210
 - **Abstract**
 Here we present the cloud population extracted from M51, following the application of our new high-resolution dust extinction technique to the galaxy (Faustino Vieira et al. 2023). With this technique, we are able to image the gas content of the entire disc of M51 down to 5 pc (0.14"), which allows us to perform a statistical characterisation of well-resolved molecular cloud properties across different large-scale dynamical environments and with galactocentric distance. We find that cloud growth is promoted in regions in the galaxy where shear is minimised; i.e. clouds can grow into higher masses (and surface densities) inside the spiral arms and molecular ring. We do not detect any enhancement of high-mass star formation towards regions favourable to cloud growth, indicating that massive and/or dense clouds are not the sole ingredient for high-mass star formation. We find that in the spiral arms there is a significant decline of cloud surface densities with increasing galactocentric radius, whilst in the inter-arm regions they remain relatively constant. We also find that the surface density distribution for spiral arm clouds has two distinct behaviours in the inner and outer galaxy, with average cloud surface densities at larger galactocentric radii becoming similar to inter-arm clouds. We propose that the tidal interaction between M51 and its companion (NGC 5195) - which heavily affects the nature of the spiral structure - might be the main factor behind this.

by olozhika (Xing Yuchen). 

2023-10-30
